docs/api.md

```diff
@@ -571,6 +574,157 @@ Content:
 
 last are set by `/lectures/:lecture-id/engagements?last=true`
 
+### <a name="login"></a>Login
+
+Login as a lecturer
+
+#### URL
+
+`/auth/login`
+
+#### Method
+
+POST
+
+#### Data Parameters
+
+##### Required
+
+* `email=[string]` Lectures registered email address.
+* `password=[string]` Hashed password using SHA256.
+
+#### Success Responses
+
+##### Success
+
+Code: 200
+
+Content:
+```
+{
+    "type": "boolean"
+    "description": "Flag for success",
+}
+```
+
+#### Error Responses
+
+##### Wrong user credentials
+
+Either the email or password or both was wrong.
+See error message for details.
+
+Code: 404
+
+Content:
+```
+{
+    "type": "object",
+    "properties": {
+        "message": {
+            "description": "Error message",
+            "type": "string"
+        }
+    }
+}
+```
+
+### <a name="check-login"></a>Check Login
+
+Check if the user is still logged id.
+
+#### URL
+
+`/auth/login`
+
+#### Method
+
+GET
+
+#### Success Responses
+
+##### Valid user token
+
+The user is still logged in.
+
+Code: 200
+
+Content:
+```
+{
+    "description": "Flag for success",
+    "type": "boolean"
+}
+```
+
+#### Error Responses
+
+##### Invalid user token
+
+The user is not logged in.
+
+Code: 404
+
+Content:
+```
+{
+    "type": "object",
+    "properties": {
+        "message": {
+            "description": "Error message",
+            "type": "string"
+        }
+    }
+}
+```
+
+
+### <a name="logout"></a>Logout
+
+Logout user
+
+#### URL
+
+`/auth/logout`
+
+#### Method
+
+GET
```

Comment:
Hm, I think it's wrong that a GET should change the state. I think it's fine with a POST without any data.